src/main/java/data/AnalysisResults.java

```diff
@@ -51,11 +53,16 @@ public String getRawText() {
     return timestamp;
   }
 
+  public Optional<Double> getPrice() {
+    return price;
+  }
+
   public static class Builder {
     private final String rawText;
     private ImmutableSet<String> categories;
     private Optional<String> store;
```

Comment:
Should this also be initialized with Optional.empty()

Reply:
Yeah, that'll be changed once I merge master in.